default/stringtables/en.txt

```diff
@@ -12582,14 +12582,14 @@ PLC_CHECKPOINTS
 Border Checkpoints
 
 PLC_CHECKPOINTS_DESC
-'''Decreases [[metertype METER_STEALTH]] of foreign ships and planets in [[metertype METER_SUPPLY]] by up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]]. Ships and planets are considered foreign if not owned by or allied to this empire. Unowned planets and monsters are also foreign.
+'''Decreases [[metertype METER_STEALTH]] of foreign ships in [[metertype METER_SUPPLY]] by up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]].
 
 The decrease depends on the time this policy is adopted, decreasing [[metertype METER_STEALTH]] by 1 per turn up to [[value PLC_CHECKPOINTS_MAX_STEALTH_MALUS]].
 
 Reduces [[metertype METER_INFLUENCE]] generation ([[value PLC_CHECKPOINTS_TARGET_INFLUENCE_FLAT]]).'''
```

Comment:
`...on planets of the empire that adopts this policy` or similar?

Reply:
I'll write per a colony if the IP cost effect is removed from outposts. Charging IP on outposts is currently running into outposts not have PP/RP/IP growth/decay and inconsistent with other policies.